src/test/librados/c_read_operations.cc

```diff
@@ -470,7 +472,8 @@ TEST_F(CReadOpsTest, Exec) {
   uint64_t features;
   EXPECT_EQ(sizeof(features), bytes_read);
   // make sure buffer is at least as long as it claims
-  ASSERT_TRUE(out[bytes_read-1] == out[bytes_read-1]);
+  std::memcpy(&features, out, bytes_read);
```

Comment:
since this is an encoded value, you'd want to add it a bufferlist and ::decode() it to guarantee it works regardless of big/little endian-ness